src/meta/TableSpec.h

```diff
@@ -185,5 +201,57 @@ struct TableSpecEqual {
 
 using TableSpecSet = nebula::common::unordered_set<TableSpecPtr, TableSpecHash, TableSpecEqual>;
 
+constexpr auto HOUR_MINUTES = 60;
+constexpr auto MINUTE_SECONDS = 60;
+constexpr auto DAY_HOURS = 24;
+constexpr auto HOUR_SECONDS = HOUR_MINUTES * MINUTE_SECONDS;
+constexpr auto DAY_SECONDS = HOUR_SECONDS * DAY_HOURS;
+
+const nebula::common::unordered_map<nebula::meta::PatternMacro, std::string> patternStr{
+  { nebula::meta::PatternMacro::DATE, "date" },
```

Comment:
feels to me - these strings should be wrapped by "{}" as "{date}", "{hour}", etc. 
otherwise, too easy to match incorrectly from a path string.

Reply:
updated in 
c612da3